.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: ruby
-rvm:
```

Comment:
This is no longer necessary because Travis reads the version from `.ruby-version`.